draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1363,112 +1242,42 @@ responding agents.
 
 <section anchor="sec-encoding" title="Encoding the Candidate Information">
 
-<t>
-Regardless of the agent being an Initiator or Responder Agent,
-the following parameters and their data types needs to be conveyed
-as part of the candidate exchange process. The specifics of syntax
-for encoding the candidate information is out of scope of this
-specification.
+<t>Agents (initiating and responding) needs the following information
+about candidates to be exchanged.  How this information is encoded or
+exchanged is out of scope of this specification.  The using protocol
+should provide a means for exchaning new, additional information in
+the future, including per-candidate information.
 </t>
 
 <t><list style="hanging">
 
-<t hangText="Candidate attribute"> There will be one or more of these
```

Comment:
I think that the using documents would benefit of having these more explicitly specified, so I would keep the original version.